External/Plugins/ASCompletion/Completion/ASComplete.cs

```diff
@@ -1344,31 +1344,94 @@ static private void ShowCalltip(ScintillaControl Sci, int paramIndex, bool force
             else UITools.CallTip.CallTipSetHlt(start + 1, end, true);
         }
 
-        static string[] featStart = new string[] { "/*", "{", "<", "[", "(" };
-        static string[] featEnd = new string[] { "*/", "}", ">", "]", ")" };
-
         static private int FindNearSymbolInFunctDef(string defBody, string symbol, int startAt)
```

Comment:
I see, yes, if wanted it could be changed, I don't know if there are chances of wanting to support something else.

Will try to commit the change later today.